packages/auth/tests/session.spec.ts

```diff
@@ -120,7 +162,7 @@ describe('Wallet integration', function () {
   it("Should dump and load a session", async () => {
     const referenceSigner = ethers.Wallet.createRandom()
 
-    const dump = (await Session.open({
+    const dump = await (await Session.open({
```

Comment:
`await (await (` is kinda hard to parse, it took me a sec to realize i was awaiting the result of `dump`. Maybe this would be easier to read as `await Session.open({}).then(s => s.dump())`